app/src/main/java/com/github/polybooks/ListSalesActivity.kt

```diff
@@ -15,7 +15,14 @@ import com.google.firebase.Timestamp
  * Activity to list all active sales
  * @property saleQuery the query listing what is to be shown
  */
+// TODO enlever parametre
 class ListSalesActivity(private val saleQuery: SaleQuery = DummySalesQuery()) : AppCompatActivity() {
+
+    companion object {
+        val EXTRA_SALE_QUERY :String = "saleQuery"
+        val EXTRA_BOOKS_QUERY : String = "bookQuery"
```

Comment:
Pq y a ceci du coup?

Reply:
those are the keys to retrieve the extra parameter (either a `SaleQuery` either `BookQuery`)